etc/smashbox.conf.template

```diff
@@ -23,6 +26,7 @@ oc_account_password=""
 # owncloud test server
 # if left blank or "localhost" then the real hostname of the localhost will be set
 oc_server = ''
+oc_root = 'owncloud'
```

Comment:
We should see how oc_root interplays with oc_webdav_endpoint. Probably oc_webdav_endpoint should be defined as "%(oc_root)s/remote.php/webdav". Also please note that oc_root (which is a part of visible URL) for many installation is just and empty string. Please check if this usage is compatible with your concept of this configuration variable.